lib/CPathfinder.cpp

```diff
@@ -193,16 +195,24 @@ void MovementCostRule::process(
 	const PathfinderConfig * pathfinderConfig,
 	CPathfinderHelper * pathfinderHelper) const
 {
-	int turnAtNextTile = destination.turn, moveAtNextTile = destination.movementLeft;
+	bool textTurn = false;
+	float costAtNextTile = destination.cost;
```

Comment:
Where is destination.cost set?

Reply:
In `CPathfinder::calculatePaths()` main loop